### DISCUSSION TITLE
The new version is a calamity

### DISCUSSION THREAD

**Discussion by Ohyoo-Labs** [picocss/pico#505] [General]
Unnecessarily complex and rebellious. Incompatible with previous projects that use it. The only thing notable is the addition of the color palette, but that does not justify destroying such a beautiful and practical library.

**Comment by asbjornu** [comment]
I just want to voice my opposition. Pico CSS 2 is a huge improvement over version 1, which was already awesome. If anything, it has gotten easier to integrate than version 1. Start from scratch, add Pico 2, then re-add the styles you feel are missing.
